content/docs/command-reference/push.md

```diff
@@ -11,8 +11,8 @@ usage: dvc push [-h] [-q | -v] [-j <number>]
                 [targets [targets ...]]
 
 positional arguments:
-  targets        Limit command scope to these DVC-files. Using -R,
-                 directories to search DVC-files in can also be given.
+  targets        Limit command scope to these `.dvc` files. Using -R,
+                 directories to search `.dvc` files in can also be given.
```

Comment:
Same as in pull:
```suggestion
  targets        Limit command scope to these stages or `.dvc` files.
                 Using -R, directories can also be given, to search
                 `.dvc` files in.
```

Reply:
@utkarshsingh99 note that this same change or a very similar one should apply to fetch and status, although those aren't included in this chunk... But maybe they should! Would you mind adding them from your original branch with all your changes (and review them to see if you can detect further improvements based on all the feedback here so far)?

Reply:
Sure. 
I checked the last commit on these files was 22 and 26 days ago, and I made that change and pulled quite later.
So I'm just going to copy the content from that branch and put it in this one for these two files.
Hope that's okay?

> note that this same change or a very similar one should apply to fetch and status

Noted. On it...

Reply:
Sure that works.